packages/expo-updates/e2e/__tests__/utils/simulator.ios.ts

```diff
@@ -2,12 +2,12 @@ import spawnAsync from '@expo/spawn-async';
 import path from 'path';
 
 const BUNDLE_IDENTIFIER = 'dev.expo.updatese2e';
-const APP_PATH = path.join(process.env.ARTIFACTS_DEST, 'ios-release.app');
 
 export const ExportedManifestFilename = 'ios-index.json';
 
-export async function installApp() {
-  await spawnAsync('xcrun', ['simctl', 'install', 'booted', APP_PATH]);
+export async function installApp(suffix: string) {
```

Comment:
```suggestion
export async function installApp(fileSuffix: string) {
```
(to match argument in other function)